lib/src/client/transport/http2_credentials.dart

```diff
@@ -30,24 +30,38 @@ typedef bool BadCertificateHandler(X509Certificate certificate, String host);
 /// certificates, etc.
 bool allowBadCertificates(X509Certificate certificate, String host) => true;
 
-class Http2ChannelCredentials extends ChannelCredentials {
+class ChannelOptions extends options.ChannelOptions {
```

Comment:
Aren't you likely to break people anyways, IMO this class naming is confusing...

Feel free to ignore :)

Reply:
I'll keep it for now.